proxy/coco/src/peer/control.rs

```diff
@@ -11,15 +11,23 @@ use crate::request;
 use super::run_state::Status;
 
 /// Requests sent to the peer.
+#[allow(clippy::pub_enum_variant_names)]
 #[derive(Debug)]
 pub enum Request {
     /// Request the current status.
     CurrentStatus(oneshot::Sender<Status>),
+    /// Request a project request.
+    GetProjectRequest(
```

Comment:
Could name them `Project` and `Projects` or `GetProject(s)`. I quite like the idea of the enum use being `Request::Project` and `Request::Projects` since it sounds like the commands. Just a suggestion anyway :)

Reply:
I do too. But the `Request` enum is not the thing as the "Project Reqests". If it would be named `Request::Projects` what would it the project be in that case. It's  a bit unfortunate that we refer to them as requests in the context of the waiting room and the commnication triggered by the peer control. #namingishard